README.md

```diff
@@ -1 +1,68 @@
-# workflow-image-scan
\ No newline at end of file
+# Reusable GitHub Workflow to Scan Images
+
+> This workflow is highly specific for a few Portworx-internal repositories - may not be suited for generic usage.
+
+The main goal this workflow is to:
+
+1. call Lacework scanner action ([lacework/lw-scanner-action](https://github.com/lacework/lw-scanner-action)) for the
+   input images,
+2. builds a scan result summary section and adds it to the end of the Actions run page,
+3. adds a comment into the pull request containing the scan results in short form.
+
+## Usage
+
+```yml
+jobs:
+  # ...
+  image-scan:
+    needs: [ other-job1, other-job2 ]
+    uses: portworx/workflow-image-scan/.github/workflows/image-scan.yml
+    with:
+      images: ${{ needs.other-job1.outputs.image }} ${{ needs.other-job2.outputs.image }}
+    secrets: inherit
+```
+
+or with explicit secrets:
```

Comment:
Might be worth pointing out when each variant is appropriate (identical vs. differently named secrets).